src/main/plugins/org.dita.base/xsl/common/output-message.xsl

```diff
@@ -25,7 +25,7 @@ See the accompanying LICENSE file for applicable license.
     <xsl:param name="msg" select="'***'"/>
     <xsl:param name="msgparams" select="''"/>    
     
-    <xsl:variable name="msgdoc" select="document('platform:/config/messages.xml')" as="document-node()?"/>
+    <xsl:variable name="msgdoc" select="document('platform:config/messages.xml')" as="document-node()?"/>
```

Comment:
Processing for custom URI schemes has changed slightly. If we use the old `platform:/config/messages.xml`, the catalog will try to resolve this to `file:/config/messages.xml`. We may want to raise this as a bug in xmlresolver.